tests/test_usm_type.py

```diff
@@ -216,10 +216,8 @@ def test_meshgrid(usm_type_x, usm_type_y):
 @pytest.mark.parametrize(
     "func,data",
     [
-        pytest.param(
```

Comment:
could you please also refresh `tests/third_party/cupy/core_tests/test_ndarray_complex_ops.py::TestConj` and `tests/third_party/cupy/math_tests/test_arithmetic.py::TestArithmeticRaisesWithNumpyInput` and `tests/third_party/cupy/math_tests/test_arithmetic.py::TestArithmeticUnary` (for last two it is needed to add `conj` and `conjugate` to parametrize, like cupy does)

Reply:
The current version of the first is the same as cupy, no modification is needed.
Other two are updated.